### DISCUSSION TITLE
Is there a process for documentation improvements?

### DISCUSSION THREAD

**Discussion by greenaar** [woodpecker-ci/woodpecker#4370] [Ask a Question]
I've seen a few items where the documentation is incorrect, incomplete, or absent.

If I go through the effort to find the solution, and it works, is there some way to get that information into the docs?  ie a setting where there's no default value, and no example of what is considered to be a valid value -- if i solve that, I write myself a note, but then what?

Do I report it as a feature/bug/other? Should I be pulling some repo, writing the documentation changes, and submitting a PR?

I've been enjoying working with WoodpeckerCI, and I think improved documentation would help in cases where the answer can only be found via google search/scouring issues by keyword, or reading discussions.

Thanks!

**Comment by pat-s** [comment]
https://woodpecker-ci.org/docs/next/development/docs

Easiest way is to issue a PR and let CI build a preview of the documentation.

Thanks for contributing, much appreciated 👍

**Comment by greenaar** [comment]
Perfect, thanks.  I'll go that route.  One final question - I've noticed the documentation is brief per topic, and there are generally a limited number of examples.  Is that the desired format, or is that just how it's turned out?

**Reply by anbraten** [reply]
> or is that just how it's turned out?

Is's mainly how things turn out. Most of the time a developer was requested to write a sentence about a new topic when they introduced it and most seem to prefer to write code and not documentation 🙈

Therefore help with the documentation is  highly appreciated.

**Reply by zc-devs** [reply]
> most seem to prefer to write code and not documentation

Yeah, that's true 😄 
Besides, a reason might be not having English skills, good enough for documentation.

**Comment by greenaar** [comment]
Perfect, thanks!

Expect pull requests shortly.

Cheers :)
